cg/cli/demultiplex/demux.py

```diff
@@ -211,6 +216,9 @@ def confirm_flow_cell_sync(context: CGConfig, source_directory: str):
     If so it creates a CopyComplete.txt file to show that that is the case."""
     target_flow_cells_dir = Path(context.flow_cells_dir)
     for source_flow_cell in Path(source_directory).iterdir():
+        if flow_cell_sync_confirmed(Path(target_flow_cells_dir, source_flow_cell.name)):

```

Comment:
More than one arg, use named args
```suggestion
        if is_flow_cell_sync_confirmed(Path(target_flow_cells_dir, source_flow_cell.name)):

```

Reply:
Actually only one arg, but will change the name 👍